samples/VSSample.Tests/VSSample.Tests.csproj

```diff
@@ -0,0 +1,191 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
>Project [](start = 1, length = 7)

It looks like you're using the legacy VS project file format. Can you please use the new .NET Standard project system, like we do with the other projects?